src/Signal.js

```diff
@@ -205,9 +205,13 @@
             if (! this.active) {
                 return;
             }
+            
+            var len = arguments.length, paramsArr = new Array(len);
```

Comment:
can you add a comment explaining why we are not using `Array#slice.call` for future reference? (just so in +6mo we still know why it was implemented like this)